test_reference.json

```diff
@@ -0,0 +1,84 @@
+{
+  "ckf-1.8-stable-mlflow-2.0-stable": {
+    "ckf":"1.8/stable",
+    "mlflow": "2.0/stable",
+    "k8s-substrates": {
+      "charmed-kubernetes": {
+        "min-supported": "1.24",
+        "max-supported": "1.29"
+      },
+      "microk8s": {
+        "min-supported": "1.25",
+        "max-supported": "1.29"
+      },
+      "aks": {
+        "min-supported": "1.27",
+        "max-supported": "1.29"
+      },
+      "eks": {
+        "min-supported": "1.24",
+        "max-supported": "1.29"
+      }
+    },
+    "juju-versions": {
+      "current": "3.4/stable",
+      "future": "3.5/stable"
+    }
```

Comment:
My reasoning was that 1.8/stable should be tested with stable versions only, but I guess the versions of juju will depend on our test plan (which we haven't defined). Do you think we should be testing it with edge as well?

Reply:
I think it makes sense in order to find out that a juju edge version is breaking our current stable bundle, before it is released. But sure, this is a discussion tighly coupled with our test plan. That being said, if we have it for the edge bundle, I think it would make more sense to have it here as well. That way, we avoid saying implicitly that we only want to test with stable versions, before defining a test plan.

Reply:
Will add it, if it is not needed, we can remove it.